watson_developer_cloud/speech_to_text_v1.py

```diff
@@ -41,6 +41,8 @@ def recognize(self, audio, content_type, continuous=None, model=None,
         Returns the recognized text from the audio input
         """
         headers = {'content-type': content_type}
+        if continuous is None:
```

Comment:
ok - maybe I should update the tests then instead of the code since the test expects to see `continuous=false` as a parameter in the query string.